src/xtgeo/xyz/points.py

```diff
@@ -102,15 +102,7 @@ class Points(XYZ):  # pylint: disable=too-many-public-methods
     def __init__(self, *args, **kwargs):
         """__init__ for Points()."""
         # instance variables listed
-        self._df = None
-        self._ispolygons = False
-        self._xname = "X_UTME"
```

Comment:
These were being overwritten in `super().__init__`

Reply:
Agree